pkg/scripts/os.go

```diff
@@ -139,11 +155,16 @@ exclude=kube*
 EOF
 {{ end }}
 
+sudo yum install -y yum-utils
+sudo yum-config-manager --add-repo=https://download.docker.com/linux/centos/docker-ce.repo
+
 sudo yum install -y --disableexcludes=kubernetes \
-	docker kubelet-{{ .KUBERNETES_VERSION }}-0 \
+	docker-ce-18.09.9-3.el7 \
```

Comment:
What do you think about adding a const along with the [Ubuntu one](https://github.com/kubermatic/kubeone/blob/3f395407b2a9c192192c3e734b2776a8f0790fc3/pkg/installer/installation/prerequisites.go#L29), so we have all Docker versions in one place and that we don't forget to bump one or another?

Reply:
This will be removed anyway in some not too far away future, when we will migrate away from repositories to direct binaries downloads, so I though, it doesn't matter.

Reply:
ok, done